frontend/webadmin/modules/gwt-common/src/main/java/org/ovirt/engine/ui/common/view/popup/OptionsPopupView.java

```diff
@@ -83,23 +93,53 @@
     @Path(value = "email.entity")
     Label email;
 
+    @UiField(provided = true)
+    @Ignore
+    public EntityModelRadioButtonEditor isHomePageDefault;
+
+    @UiField(provided = true)
+    @Path(value = "isHomePageCustom.entity")
+    public EntityModelRadioButtonEditor isHomePageCustom;
+
+    @UiField
+    InfoIcon isHomePageCustomInfo;
+
+    @UiField
+    @Path(value = "customHomePage.entity")
+    StringEntityModelTextArea customHomePage;
+
     @Inject
-    public OptionsPopupView(EventBus eventBus) {
+    public OptionsPopupView(EventBus eventBus, PlaceManager manager) {
         super(eventBus);
 
         localStoragePersistedOnServerCheckBox = new EntityModelCheckBoxEditor(Align.RIGHT);
         confirmSuspendingVmCheckBox = new EntityModelCheckBoxEditor(Align.RIGHT);
+        isHomePageCustom = new EntityModelRadioButtonEditor("homePage", Align.RIGHT); // $NON-NLS-1$
+        isHomePageDefault = new EntityModelRadioButtonEditor("homePage", Align.RIGHT);// $NON-NLS-1$
 
         initWidget(ViewUiBinder.uiBinder.createAndBindUi(this));
         ViewIdHandler.idHandler.generateAndSetIds(this);
 
         consolePublicKeyInfoIcon.setText(SafeHtmlUtils.fromString(constants.consolePublicKeyMessage()));
-        localStoragePersistedOnServerInfoIcon.setText(createTooltip());
+        localStoragePersistedOnServerInfoIcon.setText(createTooltipForGridSettings());
+        isHomePageCustomInfo.setText(createTooltipForHomePage());
+        isHomePageDefault.setLabel(
+                messages.homePageDefault("#" + // $NON-NLS-1$
+                        ((ApplicationPlaceManager) manager).getDefaultPlace().getNameToken()));
 
         driver.initialize(this);
     }
 
-    private SafeHtml createTooltip() {
+    private SafeHtml createTooltipForHomePage() {
+        SafeHtmlBuilder builder = new SafeHtmlBuilder();
+        builder.append(SafeHtmlUtils.fromString(constants.homePageFormat()));
+        builder.append(SafeHtmlUtils.fromSafeConstant(constants.lineBreak()));
+        builder.append(SafeHtmlUtils.fromSafeConstant(constants.lineBreak()));
```

Comment:
Why this is needed instead of just setting the message with double new line characters?